extrinsicParams:
  Tcr_px: '-0.006448731965984602'
  Tcr_py: '0.06913188278745684'
  Tcr_pz: '-0.15902318137562205'
  Tcr_qw: '0.5126038907261115'
  Tcr_qx: '0.5005915838307339'
  Tcr_qy: '-0.4780483430718825'
  Tcr_qz: '0.5005915838307339'
intrinsicParams:
  cx: '318.3291161258466'
  cy: '241.7577503175906'
  fx: '638.873467431243'
  fy: '642.1314594437359'
  k1: '-0.0395537355280666'
  k2: '1.2123390640395417'
  k3: '-2.550834552095233'
  p1: '0.009310871033607426'
  p2: '0.001641070593121559'
